eth2near/eth2near-block-relay-rs/src/main.rs

```diff
@@ -24,6 +24,10 @@ struct Arguments {
     /// The eth contract on Near will be initialized
     init_contract: bool,
 
+    #[clap(long, action = ArgAction::Set, default_value = "")]
+    /// The trusted block root for checkpoint for contract initialization
+    init_block_root: String,
+
```

Comment:
Please add an example on its usage to README

Reply:
Added a comment, the readme will be added after moving the init code to separate crate, and this work in progress